### PR TITLE
View all listings

### PR DESCRIPTION
**Pull request by bryceholcomb**

### CODE REVIEW

**Comment by bryceholcomb** [comment]
I had to change alot of the tests to pending, only the ones that require "items" since we changed that to listings

File: app/models/listing.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Listing < ActiveRecord::Base
+  validates :title, :description, :category_id, :max_guests,
+            :nightly_rate, :address1, :city, :state, :zip,
+            :user_id, presence: true
+  validates :shared_bathroom, :inclusion => {:in => [true, false]}
+  validates :title, uniqueness: true
+  validates_numericality_of :nightly_rate, greater_than: 0
```
Comment (bryceholcomb) [inline app/models/listing.rb]: Seriously? "Sexy" haha

File: db/seeds.rb [hunk 2]
```diff
@@ -26,178 +26,178 @@
...
 
 item13.categories << categories[4]
 item13.save
 
-item14 = Item.new(title: "Cupcakes",
-                  description: "Chocolate-Bacon Cupcakes with Dulce De Lech Frosting",
-                  price: 1100,
-                  image: open(path + "cupcakes.jpg"))
+item14 = Item.new(title: "Capital Hill House",
+                  description: "We like our politicians like we like our house, clean and liberal",
```
Comment (Lydias303) [inline db/seeds.rb]: :sparkles:

File: db/seeds.rb [hunk 3]
```diff
@@ -26,178 +26,178 @@
...
 
 item8.categories << categories[2]
 item8.save
 
-item9 = Item.new(title: "Bacon Potato Soup",
-                description: "Hot soup with a touch of bacon",
-                price: 800,
-                image: open(path + "soup.jpg"))
+item9 = Item.new(title: "Mt Rainier Cabin",
+                description: "If you like camping on a volcano, we have the place for you!",
```
Comment (Lydias303) [inline db/seeds.rb]: Much prefer this to the soup :house: lookin legit :eyeglasses:

File: db/seeds.rb [hunk 4]
```diff
@@ -26,178 +26,178 @@
...
 
 item2.categories << categories[0]
 item2.save
 
-item3 = Item.new(title: "Bacon, Egg, and Cheddar Scones",
-                 description: "Fresh cheddar cheese mixed with bacon and egg",
-                 price: 1500,
-                 image: open(path + "scones.jpg"))
+item3 = Item.new(title: "Denver Mountain Retreat",
+                 description: "The Air is thin up there, boil your eggs a couple minutes longer.",
```
Comment (Lydias303) [inline db/seeds.rb]: :sparkles:

File: db/seeds.rb [hunk 5]
```diff
@@ -26,178 +26,178 @@
...
 
 item16.categories << categories[4]
 item16.save
 
-item17 = Item.new(title: "Chocolate Brownies",
-                  description: "Smothered in vanilla icing and bacon!",
-                  price: 1000,
-                  image: open(path + "brownies.jpg"))
+item17 = Item.new(title: "Cherry Blossom B&B",
+                  description: "Stay in on our luxurious lupaddlewheeler river boats on Tidal Basin",
```
Comment (Lydias303) [inline db/seeds.rb]: :sparkles:

File: db/seeds.rb [hunk 6]
```diff
@@ -26,178 +26,178 @@
...
 
 item18.categories << categories[4]
 item18.save
 
-item19 = Item.new(title: "Chipotle Bacon Spicy Pecan Caramel Apples",
-                  description: "Spicy sweet savory crunchy tart crisp",
-                  price: 1100,
-                  image: open(path + "apples.jpg"))
+item19 = Item.new(title: "House of Cards",
+                  description: "Power is a lot like real estate. It’s all about location, location, location. The closer you are to the source, the higher your property value.",
```
Comment (Lydias303) [inline db/seeds.rb]: Im in the same boat, it makes sense to me to just make them pending until we make more changes and we are in the position to correct them. :sparkles:

File: db/seeds.rb [hunk 7]
```diff
@@ -26,178 +26,178 @@
...
 
 item19.categories << categories[4]
 item19.save
 
-item20 = Item.new(title: "Bacon Treats",
-                  description: "Chocolate covered bacon strips with sprinkles",
-                  price: 1600,
-                  image: open(path + "choc_covered.jpg"))
+item20 = Item.new(title: "The West Wing",
+                  description: "Join the Bartlet's, Leo (deceased), C. J., Josh, Toby, Donna, Charlie and Sam in this White house replica",
```
Comment (Lydias303) [inline db/seeds.rb]: :bird:

File: db/seeds.rb [hunk 8]
```diff
@@ -26,178 +26,178 @@
...
                                 }])
 
 path = 'app/assets/images/'
 
-#breakfast -----------------------------------------------------------------
-item1 = Item.new(title: "Bagel Sandwich",
-                 description: "Toasted bagel, fried egg, and bacon",
-                 price: 750,
-                 image: open(path + "bagel_sandwich.jpg"))
+#Denver -----------------------------------------------------------------
```
Comment (Lydias303) [inline db/seeds.rb]: :+1:

File: db/seeds.rb [hunk 9]
```diff
@@ -26,178 +26,178 @@
...
 item8.save
 
-item9 = Item.new(title: "Bacon Potato Soup",
-                description: "Hot soup with a touch of bacon",
-                price: 800,
-                image: open(path + "soup.jpg"))
+item9 = Item.new(title: "Mt Rainier Cabin",
+                description: "If you like camping on a volcano, we have the place for you!",
+                price: 8000,
+                image: open(path + "ex_seattle.jpg"))
```
Comment (Lydias303) [inline db/seeds.rb]: :volcano:

**Pull request merged**